client/apis/certification.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+export const getTermsAndConditions = () =>
+  axios.get(`/api/v1/certification/terms`);
```

Comment:
Certification file, etc. All similar files should be removed or stripped down to their bare minimums.